client/src/modules/Job/components/titlePage/BadgeItem.tsx

```diff
@@ -0,0 +1,12 @@
+import { Badge, Space } from 'antd';
+
+const App: React.FC = () => {
```

Comment:
your component should have the same name as your file. change `App` to `BadgeItem`